### ISSUE TITLE
Update gradle.properties

### ISSUE THREAD

**Issue by Ali-H888** [ionic-team/trapeze#198]
Hello

I'm using `"@trapezedev/configure": "^7.0.10"` and I want to update the `gradle.properties` in android

Is there an option to update the variables inside it?

**Comment by marioshtika** [comment]
Hey @Ali-H888 did you find a way to do this?

I think at some point this was possible (https://github.com/ionic-team/trapeze/issues/56) but I can not find it how to make it work.

**Comment by Ali-H888** [comment]
Hello @marioshtika, what I did was to replace the `gradle` file with one of my own
```
platforms:
    android:
       ....
        copy:
            -   src: ../configuration/android/gradle.properties
                dest: gradle.properties
```

**Comment by Ali-H888** [comment]
@marioshtika Yeah, same here. I didn't check it since a while now, but you got me interested in finding why, and I think I did :)

If you check [here](https://github.com/ionic-team/trapeze/blob/4d0a38fb9a4f95460d3de20dede3277c332361c9/packages/project/src/util/properties.ts#L11) you will find that they are passing `data` to the write file argument instead of the `serialized` string and that's what is causing the error. I tried changing it locally on my project and it worked when I replaced the data argument with the serialized variable.

**Comment by marioshtika** [comment]
Hey @Ali-H888 you were right. I also tested this on my local machine and it worked.

I created a [pull request](https://github.com/ionic-team/trapeze/pull/211) for this and hopefully someone will merge it on the main branch.

@mlynch ^

Thank you in advance.

**Comment by elsasantos** [comment]
Hi,
Just change the boolean value to a string value, like this:

platforms:
  android:
    properties:
      - file: gradle.properties
        entries: 
          android.enableJetifier: "true"

**Comment by marioshtika** [comment]
Hey @robingenz thank you for checking this 🙏
